meson.build

```diff
@@ -68,6 +68,12 @@ if get_option('split-usr') == 'auto'
 else
         split_usr = get_option('split-usr') == 'true'
 endif
+if split_usr
+        warning('\n!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n\n\n'
+                + '                    split-usr mode is going to be removed\n' +
+                '\n\n!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!')
+        run_command('sleep', '5')
```

Comment:
While I have no objections to the message itself, can you please remove the `sleep`?
This will slow down every build I / Debian/Ubunt buildds / our CI does artifically and unnecessarily.

Reply:
Yeah, the CI is going to be in split-usr for quite some time yet, so I agree the sleep is better removed - it will hurt us more than anybody else :-)

Reply:
Done.